plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -265,14 +251,14 @@ def _import_manifests(conduit, units):
         # All manifests if not specified
 
         if units is None:
-            criteria = UnitAssociationCriteria(type_ids=[Manifest.TYPE_ID])
+            criteria = UnitAssociationCriteria(type_ids=[constants.MANIFEST_TYPE_ID])
             units = conduit.get_source_units(criteria=criteria)
```

Comment:
Does the conduit still return ``AssociatedUnit`` instances or a similar old-style object? If so, I'll just observe that while this works, it will eventually need to be converted to use ``ContentUnit`` instances instead, presumably retrieved from the repo controller. We can do that later since we're in a time crunch now, but I wanted to at least make sure I'm understanding that correctly.